### PR TITLE
Crayon, lipstick, and penlight can now be inserted into your PDA

### PR DESCRIPTION
**Pull request by TheCarlSaganExpress**
Hopefully this will give the PDA a little more functionality to it.
:cl: KarlSaygen
add: You can now insert either a crayon, penlight, or lipstick into your PDA!
fix: Your pen will no longer drop on the floor if you remove it from your belt.
/:cl:

### CODE REVIEW

**Comment by jaxy15** [comment]
You forgot Meteor Pens.

**Comment by Bawhoppen** [comment]
#wallets2016

**Comment by Anonmare** [comment]
We just need a lipstick gun and we're golden

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -690,19 +691,25 @@ var/global/list/obj/item/device/pda/PDAs = list()
...
 				var/mob/M = loc
-				if(M.get_active_hand() == null)
-					M.put_in_hands(O)
-					usr << "<span class='notice'>You remove \the [O] from \the [src].</span>"
-					return
-			O.loc = get_turf(src)
+				M.put_in_hands(Pen || Crayon || Lip || Penlight || Meteorpen)
+				usr << "<span class='notice'>You remove \the [(Pen || Crayon || Lip || Penlight || Meteorpen)] from \the [src].</span>"
+				return
+			O.loc = get_turf(src) //If boths hands are full, the pen will be dropped on the floor
```
Comment (optimumtact) [inline code/game/objects/items/devices/PDA/PDA.dm]: O.forceMove

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 2]
```diff
@@ -690,19 +691,25 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	set name = "Remove Pen"
 	set src in usr
 
+	var/obj/item/weapon/pen/Pen = locate() in src
+	var/obj/item/toy/crayon/Crayon = locate() in src
```
Comment (optimumtact) [inline code/game/objects/items/devices/PDA/PDA.dm]: This some wackass code man.

Just create a single var on the PDA item  - contained_item and then let them store only one of these four types in there.

**Pull request closed without merging**